liblangutil/SemVerHandler.cpp

```diff
@@ -265,14 +272,16 @@ unsigned SemVerMatchExpressionParser::parseVersionPart()
 		{
 			c = currentChar();
 			if (v * 10 < v || v * 10 + static_cast<unsigned>(c - '0') < v * 10)
```

Comment:
By the way, since `v` is `unsigned`, this will result in different limits on different platforms. We should use a type of a fixed size here, e.g. `uint32_t`. Maybe we should even consider this a bug.

BTW, I would not change it in this PR. I guess it's actually a breaking change, even if trivial.

Reply:
Reported as #13706.